### ISSUE TITLE
source code template

### ISSUE THREAD

**Issue by laoshaw** [Microsoft/vscode#44081]
used geany as an editor before and it has a really nice feature, that I can set a template for my source code, be it a .c or .js file or any other source file, for example I can put author, copyright, license, or a few common header-file (or import module in js) into a template.c, so whenever I create a new file, I have a template to work on instead of re-trying everything, can vscode support this?

so basically you can create template.c, template.js, templpate.py etc and when you create a new file, the template file is used directly as the starting point.

**Comment by Alber70g** [comment]
This extension can do that for you: https://github.com/brpaz/vscode-file-templates-ext

**Comment by laoshaw** [comment]
works great. Thanks for the pointer
